po/fr.po

```diff
@@ -26909,847 +26871,947 @@ msgid "set the list of administrators for GROUP"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:177
+#, fuzzy
 msgid ""
 "Same as the command --fingerprint but changes only the format of the output "
 "and may be used together with another command"
 msgstr ""
+"Pareil que la commande --fingerprint mais ne changer que le format de "
+"sortie ; peut être utilisé avec une autre commande"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:178
 msgid "Changes the output of the list commands to work faster"
-msgstr ""
+msgstr "Change la sortie des commandes de listage pour travailler plus vite"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:179
 msgid ""
 "Do not merge primary user ID and primary key in --with-colon listing mode "
 "and print all timestamps as UNIX timestamps"
 msgstr ""
+"Ne pas fusionner l’identifiant utilisateur primaire et la clé primaire dans "
+"les listes avec --with-colon, et afficher tous les horodatages au format UNIX"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:180
 msgid ""
 "Changes the behaviour of some commands. This is like --dry-run but different"
 msgstr ""
+"Change le comportement de certaines commandes ; similaire à, quoique "
+"différent de, --dry-run"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:181
 msgid "Display the session key used for one message"
-msgstr ""
+msgstr "Afficher la clé de session utilisée pour un message"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:182
 msgid "Don't use the public key but the specified session key"
-msgstr ""
+msgstr "Utiliser la clé de session spécifiée au lieu de la clé publique"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:183
 #: /tmp/fish/implicit/share/completions/gpg.fish:185
 msgid "Prompt for an expiration time"
-msgstr ""
+msgstr "Demander une date d’expiration"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:184
 #: /tmp/fish/implicit/share/completions/gpg.fish:186
 msgid "Do not prompt for an expiration time"
-msgstr ""
+msgstr "Ne pas demander une date d’expiration"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:187
 msgid ""
 "Don't look at the key ID as stored in the message but try all secret keys in "
 "turn to find the right decryption key"
 msgstr ""
+"Ne pas rechercher l’identifiant de clé tel qu’indiqué dans le message ; "
+"essayer à la place toutes les clés privées pour trouver la bonne"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:188
 msgid ""
 "Enable a mode in which filenames of the form -&n, where n is a non-negative "
 "decimal number, refer to the file descriptor n and not to a file with that "
 "name"
 msgstr ""
+"Activer un mode dans lequel les noms de fichiers au format -&n, pour tout n "
+"entier naturel, font référence au descripteur de fichier n et pas à un "
+"fichier ainsi nommé"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:189
 msgid "Sets up a named group, which is similar to aliases in email programs"
 msgstr ""
+"Paramètre un groupe nommé, similaire aux alias pour les programmes d’email"
```

Comment:
Inconsistent terminology for email: mostly, “couriel” is used (which is more often found in _Québécois_ than in metropolitan French AFAIK).

Reply:
Froggies will not really use *courriel*; our government tried to make us use this translation instead of simply using *email*, but the last one was already too widely used to be replaced, so, if the dominant usage in France is the reference, *email* is to be used.

Reply:
I'd be all in favour of using _email_ everywhere; wouldn’t replacing _courriel_ just be one straight search and replace?

Reply:
Done, but it modified almost only ~~crappy~~ fuzzy translations, i.e. translations whose meaning is very different from the original.

Reply:
Yes, indeed, but not this time.